### ISSUE TITLE
[feat]: add gatsby-transformer-pdf

### ISSUE THREAD

**Issue by DSchau** [gatsbyjs/gatsby#8711]
## Summary

Gatsby is all about "bring your own content," and PDF is certainly a means of embedding useful data and content. Let's create a gatsby-transformer-pdf transformer plugin to enable usage of PDF as a content source, and make it queryable with Gatsby's GraphQL layer.

### Motivation

- Most common use case will be to make searchable, indexable web sites with Gatsby powered by PDFs as a document store
- To enable this, we'll need to find some way to translate PDF to a usable format, e.g. JSON
    - Haven't used it, but consider [pdf2json](https://github.com/modesty/pdf2json)

### Further Detail

Consider looking at existing transformer plugins for inspiration, e.g. [gatsby-transformer-json](https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-transformer-json), [gatsby-transformer-csv](https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-transformer-csv), etc.

**Comment by AlexMunoz** [comment]
can i work in this issue?

**Comment by DSchau** [comment]
@AlexMunoz I would love nothing more! Let us know if we can help with anything :)

**Comment by LaRuaNa** [comment]
@AlexMunoz can i help you with anything to speed up the process? :) Need ASAP this feature :)

**Comment by DSchau** [comment]
Fixed in #8817 🎉 

Thanks so much for the PR @AlexMunoz 💪

```
Successfully published:
 - gatsby-transformer-pdf@1.0.11
 - gatsby-transformer-yaml@2.1.3
```

bumped the version due to a little hiccup with our build process!
